README.md

```diff
@@ -3082,15 +3082,31 @@ The test file pattern and exclude pattern support any glob pattern handled by [`
 
 #### How to run a specific list of test files or only some tests from test file?
 
-:information_source: If you don't want to use the pattern [`KNAPSACK_PRO_TEST_FILE_PATTERN`](#how-can-i-run-tests-from-multiple-directories) to define a list of tests to run then read below.
+:information_source: If you don't want to use the pattern [`KNAPSACK_PRO_TEST_FILE_PATTERN`](#how-can-i-run-tests-from-multiple-directories) to define a list of tests to run then read below two options.
+
+**Option 1:**  
 
 If you want to run a specific list of test files that are explicitly defined by you or auto-generated by some kind of script you created then please use:
 
 `KNAPSACK_PRO_TEST_FILE_LIST=spec/features/dashboard_spec.rb,spec/models/user.rb:10,spec/models/user.rb:29`
 
 Note `KNAPSACK_PRO_TEST_FILE_LIST` must be a list of test files comma separated. You can provide line number for tests inside of spec file in case of RSpec (this way you can run only one test or a group of tests from RSpec spec file). You can provide the same file a few times with different test line number.
 
-Note when you set `KNAPSACK_PRO_TEST_FILE_LIST` then below environment variables are ignored:
+**Option 2:** 
+
+Similarly, you can also provide a source file containing the test files that you would like to run. For example:
+`KNAPSACK_PRO_TEST_FILE_LIST_SOURCE_FILE=spec/fixtures/test_file_list_source_file.txt`  
+And the content of the source file can be any of the format below:
+```
+./spec/test1.rb
```

Comment:
Maybe we could use naming with rspec default convention `_spec.rb`:

`spec/test1_spec.rb` instead of `spec/test1.rb`. 

And update paths in other examples like `spec/fixtures/test_file_list_source_file.txt` and in the `spec/knapsack_pro/test_file_finder_spec.rb`.